rules/windows/defense_evasion_defender_exclusion_via_powershell.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2021/07/20"
+maturity = "production"
+updated_date = "2021/07/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies modifications to the Windows Defender configuration settings using PowerShell to add exclusions at the folder directory or process level.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Windows Defender Exclusions Added via PowerShell"
+note = """## Triage and analysis
+
+Detections should be investigated to identify if the activity corresponds to legitimate activity used to put in exceptions for Windows Defender. As this rule detects post-exploitation process activity, investigations into this should be prioritized"""
+references = ["https://www.bitdefender.com/files/News/CaseStudies/study/400/Bitdefender-PR-Whitepaper-MosaicLoader-creat5540-en-EN.pdf"]
+risk_score = 21
+rule_id = "2c17e5d7-08b9-43b2-b58a-0270d65ac85b"
+severity = "low"
```

Comment:
should we up it? @brokensound77?

Reply:
yea definitely, to at least `medium`. If we had a better sense of this occurring in an expected matter it could even justify `high`

Reply:
```suggestion
severity = "medium"
```